julia/core.py

```diff
@@ -286,7 +289,11 @@ def __init__(self, init_julia=True, jl_init_path=None):
                     "Install PyCall by running the following line:\n"
                     """\tjulia -e 'Pkg.add("PyCall"); Pkg.update()'\n""")
             try:
-                self.call('pyinitialize(C_NULL)')
+                if os.name == "nt":
+                    # on windows C_NULL doesn't mean anything
+                    self.call('pyinitialize()')
```

Comment:
more precisely, a variable -1 is asked to `Py_GetVersion` , which is clueless about its meaning.
I may be wrong on the true cause, but the fix works